src/test/java/com/purbon/kafka/topology/integration/containerutils/KsqlContainer.java

```diff
@@ -7,7 +7,7 @@
 public class KsqlContainer extends GenericContainer<KsqlContainer> {
 
   private static final DockerImageName DEFAULT_IMAGE =
-      DockerImageName.parse("confluentinc/ksqldb-server").withTag("0.21.0");
+      DockerImageName.parse("confluentinc/ksqldb-server").withTag("0.26.0");
```

Comment:
For consistency this should be rather confluentinc/cp-ksqldb-server with tag same as other CP components.
Again, we can tackle it separately.

Reply:
I agree to that too. :-)